### PR TITLE
Ha mode

### PR DESCRIPTION
**Pull request by nikolay-fedotov**
Tests for HA mode

### CODE REVIEW

File: tests/modules/test_deployment_ha_mode.py [hunk 1]
```diff
@@ -1,72 +1,92 @@
...
+
+
+class Test_Deployment_HA_Mode(TestCasePoteen):
+    @classmethod
+    def setUpClass(cls):
+        super(Test_Deployment_HA_Mode, cls).setUpClass()
+        PoteenLogger.add_test_suite("Cluster deployment")
+        ContextHolder.set_browser("firefox")
+        ContextHolder.set_do_screenshot(False)
+        ContextHolder.set_url("http://10.20.0.2:8000/")
```
Comment (nikita-prikazchikov) [inline tests/modules/test_deployment_ha_mode.py]: Please comment these lines on commits. this is debug only code.

File: tests/components/constants.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 class TestConstants(object):
 
-    OPENSTACK_FOLSOM = "Folsom"
+    OPENSTACK_FOLSOM = "Grizzly"
```
Comment (nikita-prikazchikov) [inline tests/components/constants.py:5]: Add constant OPENSTACK_GRIZZLY and change OPENSTACK_CURRENT_VERSION to OPENSTACK_GRIZZLY

Comment (e0ne) [inline tests/components/constants.py:5]: Don't create a class to store only constants

**Comment by nikolay-fedotov** [comment]
Should we comment ContextHolder.set functions calls in all SetUp functions everywhere?

**Comment by KseniiaCherchenko** [comment]
+1 verify on fake ui of master branch

**Pull request merged**